### PR TITLE
Feature/14128 ess u i accessibility testing for exchange sets download page

### PR DESCRIPTION
**Pull request by Rajesh14881**
Accessibility testing for arc tool ,site improve and invalid enc  show in ukho dialogue

### CODE REVIEW

**Review by JiviteshT — APPROVED** [review]
Make sure code coverage is enough otherwise looks fine.

File: src/app/features/exchange-set/ess-download-exchangeset/ess-download-exchangeset.component.scss [hunk 1]
```diff
@@ -28,9 +32,21 @@
     margin: 0px 39px;
 }
 
-
 .f22{
```
Comment (rvijayavel) [inline src/app/features/exchange-set/ess-download-exchangeset/ess-download-exchangeset.component.scss:35]: Please check the class name and font-size.  And for .f36 class also.

**Review by rvijayavel — APPROVED** [review]
Only css class name and font-size needs to be verified, apart from that all looks good.

File: src/app/features/exchange-set/ess-download-exchangeset/ess-download-exchangeset.component.scss (hunk 1 above)
Comment (Rajesh14881) [reply src/app/features/exchange-set/ess-download-exchangeset/ess-download-exchangeset.component.scss]: resolved

**Pull request merged**